hail/python/hail/methods/statgen.py

```diff
@@ -2013,7 +2018,7 @@ def error_on_moved(v):
                         .or_error("Found non-left-aligned variant in split_multi"))
             return hl.bind(error_on_moved,
                            hl.min_rep(old_row.locus, [old_row.alleles[0], old_row.alleles[i]]))
-        return split_rows(hl.sorted(kept_alleles.map(make_struct)), False)
+        return split_rows(hl.sorted(kept_alleles.map(make_struct)), permit_shuffle)
```

Comment:
I'm not *too* sure it makes sense to allow both `permit_shuffle` and `left_aligned` here, since the `left_aligned` flag is kind of a performance thing to prevent the more expensive shuffle/join, but I also don't have strong feelings about keeping it like this.